x-pack/legacy/plugins/siem/server/kibana.index.ts

```diff
@@ -72,12 +72,19 @@ export const initServerWithKibana = (
     catalogue: ['siem'],
     privileges: {
       all: {
-        api: ['siem'],
+        api: ['siem', 'actions-read', 'actions-all', 'alerting-read', 'alerting-all'],
         savedObject: {
-          all: [noteSavedObjectType, pinnedEventSavedObjectType, timelineSavedObjectType],
+          all: [
+            'alert',
+            'action',
+            'action_task_params',
+            noteSavedObjectType,
+            pinnedEventSavedObjectType,
+            timelineSavedObjectType,
+          ],
           read: ['config'],
```

Comment:
Will SIEM have read access to alerts when `read` privileges are granted?

Reply:
I'd assume so, yes.

Reply:
@FrankHassanabad I'm working on adding privileges to alerting consumers. For SIEM's case, did you want read only access to alerting when `read` is granted for SIEM? or no alerting access at all?

Reply:
I would say `all` since we are creating and doing reading with alerts on the detection engine page.